template/FullStack/Vite(Frontend)+Flask(Backend)/server/app.py

```
@@ -0,0 +1,30 @@
+from flask import Flask
+from flask_cors import CORS
+from pymongo import MongoClient
+from routes import auth_routes 
+from dotenv import load_dotenv
+import gunicorn
```

Ruff

`gunicorn` imported but unused

Remove unused import: `gunicorn`

(F401)